build/Targets/Imports.targets

```diff
@@ -245,6 +249,16 @@
     </SuggestedBindingRedirects>
   </ItemGroup>
 
+  <!-- 
+       Work around some UI delays in the new project system
+
+       https://github.com/dotnet/project-system/issues/2297
```

Comment:
This bug is already closed. Can the comment be updated to state what criteria we're still waiting for on this one?